io.blockloader.BlockLoader.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.blockloader.BlockLoader
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: blockloader
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+modules:
+  - name: blockloader
+    buildsystem: simple
+    build-commands:
+      - mv BlockLoader-linux-x64 /app/blockloader
+      - chmod +x /app/blockloader/blockloader
+      - install -Dm755 run.sh /app/bin/blockloader
+      - install -Dm644 -t /app/share/metainfo io.blockloader.BlockLoader.metainfo.xml
+      - install -Dm644 logo.png /app/share/icons/hicolor/256x256/apps/io.blockloader.BlockLoader.png
+      - install -Dm644 -t /app/share/applications io.blockloader.BlockLoader.desktop
+    sources:
+      - type: archive
+        url: https://cdn.blockloader.io/releases/linux/x64/BlockLoader-linux-x64-0.5.6.zip
+        sha256: 27a76b9fa249079d7e14537159b81a35b30064b321184bcfbed2a2d650587e75
+        strip-components: 0
+        x-checker-data:
+          type: html
+          url: https://blockloader.io
+          version-pattern: <span>v([\\d\\.-]*)</span>
+          url-template: https://cdn.blockloader.io/releases/linux/x64/BlockLoader-linux-x64-$version.zip
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/blockloader/blockloader "$@"
+      - type: file
+        path: io.blockloader.BlockLoader.metainfo.xml
+      - type: file
+        url: https://cdn.blockloader.io/logo.png
+        sha256: cf439d5568c0803b5000cd0a232442f9b5297b94f919cd22991957a13e241979
```

Comment:
Are you sure this File doesn't change?

Reply:
It's unlikely that it will change frequently. 

Would it be better to include a copy of the logo in this repo instead?

Reply:
it would be better in the tarball. I mean you ship a desktop app without an icon?

Reply:
(and also without a .desktop)

Reply:
It's an Electron app, so I'm using Electron Forge with the zip maker to build the release. I'm using Flathub as a convenient way to bring the app to Linux, so I don't really have an existing Linux desktop application.